projects/kit/components/input-date-multi/input-date-multi.template.html

```diff
@@ -0,0 +1,79 @@
+<tui-hosted-dropdown
+    class="t-hosted"
+    [canOpen]="canOpen"
+    [content]="dropdown"
+    [open]="open && canOpen"
+    (openChange)="onOpenChange($event)"
+>
+    <tui-input-tag
+        *tuiLet="filler$ | async as filler"
+        #inputTag
+        automation-id="tui-input-date-tag__textfield"
```

Comment:
Let's not add `automation-id` if can. In the future we should drop them all in favor of page objects/harnesses.

Reply:
fixed